packages/next/README.md

```diff
@@ -1337,6 +1337,40 @@ module.exports = {
 }
 ```
 
+#### Configuring ESM to use import/export statements in next.config.js
```

Comment:
Let's remove this part and actually document how to pass node.js arguments instead 👍